packages/cyberstorm/src/components/componentStyles/Tag.module.css

```diff
@@ -17,46 +17,49 @@
   --border-color: transparent;
 }
 
-.tag:hover {
+.root:hover {
   --bg-color: var(--color-surface--0);
   --border-color: var(--color-border-3);
 }
 
 /* Different Tag sizes */
 
 .small {
-  gap: var(--size-tiny);
-  padding: var(--size-tiny) var(--text-xxs);
-  font: normal 700 var(--text-xxs) Inter, sans-serif;
-  line-height: var(--size-tiny);
+  gap: var(--gap-tiny);
+  padding: var(--padding-tiny) var(--padding-xxs);
+  font: var(--font-inter);
```

Comment:
You can probably move the `font`and `font-weight`properties to `.root` as they don't change between the size variants to avoid repetition.

Reply:
Nice catch👍